File: ruby/arcade/electionsWinners.rb
def electionsWinners(votes, k)
    winners, len = 0, votes.size
    len.times do |i|
        isWinning = true
        len.times do |j|
            if j != i and not(votes[i] + k > votes[j]) then
                isWinning = !isWinning
                break
            end
        end
        winners += 1 if isWinning
    end
    return winners
end

File: java/arcade/ElectionsWinners.java
public final class ElectionsWinners {

	int electionsWinners(int[] votes, int k) {
		int winners = 0, len = votes.length;
		boolean isWinning = true;
		for(int i = 0; i < len; i++, isWinning = true) {
			for(int j = 0; j < len; j++) {
				if(j != i && !(votes[i] + k > votes[j])) {
					isWinning = !isWinning;
					break;
				}
			}
			if(isWinning) { winners++; }
		}
		return winners;
	}
}